src/HasSize.php

```
@@ -0,0 +1,87 @@
+     *
+     * Usage example:
+     * ```php
+     * $element->size(10);
+     * $element->size(50);
+     * $element->size(null);
+     * ```
+     */
+    public function size(int|string|Stringable|UnitEnum|null $value): static
+    {
+        if ($value instanceof UnitEnum) {
+            $value = Enum::normalizeValue($value);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Enum' in method 'size'. (undefined)

(StaticAccess)